.github/workflows/testing.yml

```diff
@@ -45,4 +45,4 @@ jobs:
       run: |
         pip install torchtestcase
         pip install pytest
-        pytest
+        pytest -m "not slow"
```

Comment:
Perhaps it's better to be explicit and use the reverse, i.e. mark tests as 'fit for CI' or even 'CI'. Then one would run pytest -m 'CI'. It feels like this is more directly descriptive of what we want. We can discuss this next time, though (i.e. not a blocker for this PR).

Reply:
I find the slow marker more intuitive than just CI, plus marking all fasts tests is more work than marking just slows tests. but we can discuss in a separate PR.